config/playground/helm/prod/helm-opensearch-dashboards.yaml

```diff
@@ -185,4 +185,12 @@ extraObjects: []
 # specify the external plugins to install
 plugins:
   enabled: true
-  installList: ["https://github.com/BionIT/google-analytics-plugin/releases/download/2.18.0/googleAnalytics-2.18.0.zip"]
\ No newline at end of file
+  installList: ["https://github.com/BionIT/google-analytics-plugin/releases/download/2.18.0/googleAnalytics-2.18.0.zip"]
```

Comment:
Usually, we do not enable dev and prod at the same time. We need to enable dev first, verify it works, and then enable prod. Can this PR enable only dev for now? After verification, we can enable prod with another PR. Thanks!

Reply:
It's not hurry. Let's merge this PR after I've verified it locally in **dev**.

Reply:
If you take a look the github action in this repo, it is monitoring the file changes from folder, https://github.com/opensearch-project/dashboards-anywhere/blob/main/.github/workflows/os-osd-deployment.yml#L26, you need to update your PR only have dev file changes, it will trigger dev env deployment, then you can verify if your changes works. after everything works expected, then create another PR for modify the PROD file.